app/screens/reactions/reactors_list/index.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useEffect, useRef, useState} from 'react';
+import {NativeScrollEvent, NativeSyntheticEvent, PanResponder} from 'react-native';
+import {FlatList} from 'react-native-gesture-handler';
+
+import {fetchUsersByIds} from '@actions/remote/user';
+import {useServerUrl} from '@context/server';
+
+import Reactor from './reactor';
+
+import type ReactionModel from '@typings/database/models/servers/reaction';
+
+type Props = {
+    reactions: ReactionModel[];
+}
+
+const ReactorsList = ({reactions}: Props) => {
+    const serverUrl = useServerUrl();
+    const [enabled, setEnabled] = useState(false);
+    const listRef = useRef<FlatList>(null);
+    const [direction, setDirection] = useState<'down' | 'up'>('down');
```

Comment:
Nit: Can we move this line a bit higher so all `useState` are together and all `useRef` are together?